Uppsala/UppQ1524o.xml

```diff
@@ -0,0 +1,627 @@
+                                    ብእሲ፡ ኢይጻእ፡ እምአንቀጸ፡ ቤቱ፡ እስመ፡ አረጋዊ፡ ውእቱ፡ ዘወጽአ፡ እምነ፡ ገነት፡ ንድቅ፤
+                                </incipit>
+                                <note>In the upper left corner of the folio, the following note has been written: 
+                                    "G.L."</note>
+                            </msItem>
+                            
+                            
+                            <msItem xml:id="p1_i18">
+                                <locus target="#36r #37r"/>
+                                <title type="complete" ref="LIT7075QeneHagarena"/>
+                                <incipit xml:lang="gez">
+                                    ሐገርነ፡ ኮነት፡ ከመ፡ ሰማርያ፡ እስመ፡ በላእሌሃ፡ ወረደ፡ ነጠብጣብ፡ ዝናም፡ መዓት፤
+                                </incipit>
+                                <note>In the upper left corner of the folio, the following note has been written: 
+                                    "G.L."</note>
+                            </msItem>
+                            
+                            
+                            <msItem xml:id="p1_i19">
+                                <locus target="#38r #39r"/>
+                                <title type="complete" ref="LIT7073QeneTamayta"/>
+                                <incipit xml:lang="gez">
+                                    ተመይጠት፡ ጽዮን፡ ጼዋ፡ አጵሎን፡ ነኪር፤
+                                </incipit>
+                                <note>In the upper left corner of the folio, the following note has been written: 
+                                    "G.L."</note>
+                            </msItem>
+                            
+                            
+                            <msItem xml:id="p1_i20">
+                                <locus target="#40r #41r"/>
+                                <title type="complete" ref="LIT7072QeneMamhera"/>
+                                <incipit xml:lang="gez">
+                                    መምህረ፡ ዝንቱ፡ ዓለም፡ አውስጣቴዎስ፡ ደብረ፡ አርማንያ፡ ወልደከ፡ በጽሐ፡ ለዘምሮ፤
+                                </incipit>
+                                <note>In the upper left corner of the folio, the following note has been written: 
+                                    "G.L."</note>
+                            </msItem>
+                            
+                            
+                            <msItem xml:id="p1_i21">
+                                <locus target="#42r"/>
+                                <title type="complete" ref="LIT7099QeneEffoHalloka"/>
+                                <incipit xml:lang="gez">
+                                    <choice><sic>አ</sic><corr resp="JK">እ</corr></choice>ፎ፡ ሀሎከ፡ በጊዜ፡ 
+                                    ትቤ<del rend="overUnderlined strikethrough">ሳ</del><add place="above">ላ</add>፡ ኦሪተ፡ ሙሴ፡ ቅዱስ፤
+                                </incipit>
+                                <note>In the upper left corner of the folio, the following note has been written: 
```

Comment:
Can you spekulate who added 'G.L'? If yes, I think you can say it. If you have reasons to think that it was not anyone we expect, may be you need to explain the hand.

Reply:
I think it is probably by the scribe of the main text, which I suspect is Kolmodin. I will add it!